css/components/alerts.css

```diff
@@ -0,0 +1,100 @@
+.alert-container {
+  position: fixed;
+  top: 1rem;
+  right: 0;
+  z-index: 4;
+}
+
+.alert-container.hide-display {
+  display: none;
+}
+
+.alert-icon {
+  width: 2.4rem;
+  height: 2.4rem;
+}
+
+.alert {
+  border-radius: 0.5rem;
+  max-width: 60rem;
+  width: 100%;
+  font-size: 1.8rem;
+  display: flex;
+  align-items: center;
+  justify-content: start;
+  padding: 2rem 4rem;
+  gap: 1rem;
+  color: white;
+  border-radius: 0.5rem;
+  box-shadow: var(--box-shadow-bottom);
+}
+
+.alert-info {
+  background-color: var(--info-color-tint);
+  border-left: 6px solid var(--info-color);
+  color: var(--info-color);
+}
+
+.alert-info .close-btn-icon {
+  color: var(--info-color);
+}
+
+.alert-warning {
+  background-color: var(--warning-color-tint);
+  border-left: 6px solid var(--warning-color);
+  color: var(--warning-color);
+}
+
+.alert-warning .close-btn-icon {
+  color: var(--warning-color);
+}
+
+.alert-success {
+  background-color: var(--success-color-tint);
+  border-left: 6px solid var(--success-color);
+  color: var(--success-color);
+}
+
+.alert-success .close-btn-icon {
+  color: var(--success-color);
+}
+
+.alert-error {
+  background-color: var(--error-color-tint);
+  border-left: 6px solid var(--error-color);
+  color: var(--error-color);
+}
+
+.alert-error .close-btn-icon {
+  color: var(--error-color);
+}
```

Comment:
Instead of making separate classes only for the close-btn-icon you can create general utility classes named as --kui-error-color{
//color description
}
and use it in the HTML files along with other types too such as warning info